src/libuv/Native.Settings

```diff
@@ -50,6 +50,8 @@
     <Link>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
       <OptimizeReferences>true</OptimizeReferences>
+      <ControlFlowGuard>Guard</ControlFlowGuard>
```

Comment:
We only ship release bits I believe, and I wanted to keep `/ZI` in debug for better symbols if building and running locally. And this switch is not compatible with `/ZI`